src/store/backendApi.ts

```diff
@@ -0,0 +1,6 @@
+import { useGetBlueprintsQuery as useImageBuilderGetBlueprintsQuery } from './imageBuilderApi';
+import { useGetBlueprintsQuery as useCockpitGetBlueprintsQuery } from './cockpitApi';
+
+export const useGetBlueprintsQuery = process.env.IS_ON_PREMISE
```

Comment:
Again, same concept here, we can just import `useGetBlueprints` query from this file.
And then depending on what environment we're in, we should get the one we need.

Reply:
I love this, it's so simple.

Reply:
Thanks 😄